lib/features/family/features/reflect/presentation/pages/stage_screen.dart

```diff
@@ -18,26 +18,29 @@ class StageScreen extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return FunScaffold(
-      withSafeArea: false,
-      appBar: const FunTopAppBar(
-        title: '',
-        actions: [LeaveGameButton()],
-      ),
-      body: Container(
-        decoration: const BoxDecoration(
-          color: FamilyAppTheme.primary99,
-          image: DecorationImage(
-            image: AssetImage('assets/family/images/stage_background.png'),
-            fit: BoxFit.fitWidth,
+    return PopScope(
+      canPop: false,
+      child: FunScaffold(
+        withSafeArea: false,
```

Comment:
the funscaffold has canPop param, please use that

Reply:
thanks, i missed that, fixed it!